core/spec/lib/spree/core/validators/email_spec.rb

```diff
@@ -16,8 +22,19 @@ class Tester
     'valid-email@email.com',
     'valid_email@email.com',
     'validemail_@email.com',
-    'valid.email@email.com'
+    'valid.email@email.com',
```

Comment:
@Tenzil Yes, I checked it works with above email, if you want you can fork this branch and run test at your local.